Model/Model/IndexedModel.php

```diff
@@ -128,18 +128,18 @@ public function getIndexDocumentData()
                     break;
                 } elseif ($results instanceof Collection) {
                     $nestedData = $results->map(function (IndexedModel $result) {
-                        return array_intersect_key($result->attributesToArray(), $result->mappingProperties);
+                        return array_intersect_key($result->attributesToArray(), $result->getMappingProperties());
```

Comment:
@jeremysik `\Elasticquent\ElasticquentTrait::getMappingProperties` is just getter for that property